Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>CSU_SET_PASSWORD = 'viva416384352'\r\nEMAIL_HOST_USER = 'andrey01590@gmail.com'\r\n\r\nDATABASES_PASSWORD= aktiv416384352\r\nSECRET_KEY = 'django-insecure-ak!0$hnt2e!^*u12ju&@$s_$88&ma0h+_x)fl!pzh94n-^902#'
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env b/.env
--- a/.env	(revision cd34eb94e8f1a8d2f44a68090835abe14b2eb0a8)
+++ b/.env	(date 1705914688306)
@@ -2,4 +2,4 @@
 EMAIL_HOST_USER = 'andrey01590@gmail.com'
 
 DATABASES_PASSWORD= aktiv416384352
-SECRET_KEY = 'django-insecure-ak!0$hnt2e!^*u12ju&@$s_$88&ma0h+_x)fl!pzh94n-^902#'
\ No newline at end of file
+SECRET_KEY = 'django-insecure-ak!0$hnt2e!^*u12ju&@$s_$88&ma0h+_x)fl!pzh94n-^902#'
